examples/webgl_loader_gltf_anisotropy.html

```diff
@@ -53,12 +53,13 @@
 
 				scene = new THREE.Scene();
 
-				camera = new THREE.PerspectiveCamera( 50, window.innerWidth / window.innerHeight, 0.25, 20 );
+				camera = new THREE.PerspectiveCamera( 50, window.innerWidth / window.innerHeight, 0.02, 2 );
```

Comment:
I would avoid setting small values of the near plane.

Instead, I would do this:
```js
controls.enableZoom = false;
controls.enablePan = false;
```

There is no benefit here to zooming out.

Reply:
> There is no benefit here to zooming out.

But there sure is benefit to zooming in 🤓

![Screenshot 2023-05-16 at 12 13 00 PM](https://github.com/mrdoob/three.js/assets/97088/a2a91f70-7449-48e1-aee3-ac5c5b7111a3)

I think it's cool to be able to see the model up close.

I'll tweak the example.